Colours_And_Fonts_style_v1.css

```diff
@@ -198,17 +288,24 @@ ul li:hover ul {
 
 .content{
 	box-sizing: border-box;
-	background-color: #219ebc;
+	background-color: #FFFFFF;
 	position: relative;
 	width: 1990px;
 	height: 1300px;
 	font-size: 12px;
 	text-align: left;
+    flex-direction: column;
+    align-items: center;
+    padding: 20px;
+    margin: auto;
+}
+
+.logo1 {
+	float: right;
 }
 
 img {
-	margin-left: 20px;
-	margin-right: 10px;
+  margin: 10px; /* Adjust the value as needed */

```

Comment:
I deleted margin-left and margin-right and replaces it with margin so it can add spacing between each image.